Formula/mongodb-enterprise.rb

```diff
@@ -0,0 +1,104 @@
+class MongodbEnterprise < Formula
+  desc "High-performance, schema-free, document-oriented database (Enterprise)"
+  homepage "https://www.mongodb.com/"
+
+  # frozen_string_literal: true
+  #
+  if Hardware::CPU.intel?
+    url "https://downloads.mongodb.com/osx/mongodb-macos-x86_64-enterprise-6.0.0.tgz"
+    sha256 "21adee4e595642101d54b5572003b7b26e72161edde16a102a89500c8e405d72"
+  else
+    url "https://downloads.mongodb.com/osx/mongodb-macos-arm64-enterprise-6.0.0.tgz"
+    sha256 "919e766690e276cb902269a178b55c1a61380b2069e68e5c9dd02d24b39ac1f1"
+  end
+
+  license "MongoDB Customer Agreement"
+
+  def caveats
+    <<~EOS
+      MongoDB Enterprise is licensed under the MongoDB Customer Agreement (https://www.mongodb.com/customer-agreement). Except for evaluation purposes, you may not use MongoDB Enterprise without a commercial license from MongoDB.
+    EOS
+  end
+
+  option "with-enable-test-commands", "Configures MongoDB to allow test commands such as failpoints"
+
+  depends_on "mongodb-database-tools" => :recommended
```

Comment:
if we add enterprise formula, should there be conflicts_with to the community package? Would they not conflict in some way? Can there be side by side installations of the two?